instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-jersey-2.0/javaagent/jaxrs-2.0-jersey-2.0-javaagent.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
   testLibrary group: 'io.dropwizard', name: 'dropwizard-testing', version: '0.8.0'
   testImplementation group: 'javax.xml.bind', name: 'jaxb-api', version: '2.2.3'
   testImplementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-afterburner', version: '2.9.10'
+
+  latestDepTestLibrary group: 'org.glassfish.jersey.core', name: 'jersey-server', version: '2.+'
+  latestDepTestLibrary group: 'org.glassfish.jersey.containers', name: 'jersey-container-servlet', version: '2.+'
```

Comment:
```suggestion
  latestDepTestLibrary group: 'org.glassfish.jersey.containers', name: 'jersey-container-servlet', version: '2.+'
  latestDepTestLibrary group: 'org.eclipse.jetty', name: 'jetty-webapp', version: '9.+'
```

Reply:
`dropwizard-testing` depends on jetty-webapp in `0.8.0` but does not depend on it in the latest version which results in a horrible mess. Might as well remove
```
testImplementation group: 'org.eclipse.jetty', name: 'jetty-webapp', version: '8.0.0.v20110901'
```
or replace it with
```
testImplementation group: 'org.eclipse.jetty', name: 'jetty-webapp', version: '9.2.9.v20150224'
```
as this is the version that it really runs with.